app/auth/middleware.test.js

```diff
@@ -73,15 +68,28 @@ describe('Authentication middleware', function() {
       context('when the user locations lookup fails', function() {
         beforeEach(async function() {
           const authentication = proxyquire('./middleware', {
-            '../../common/lib/user': UserStub,
-            '../../common/services/user-locations': userLocationFailureStub,
+            '../../common/services/user': userLocationsFailureStub,
+          })
+
+          await authentication.processAuthResponse()(req, {}, nextSpy)
+        })
+
+        it('returns next', function() {
+          expect(nextSpy).to.be.calledOnceWithExactly(userFailureError)
+        })
+      })
+
+      context('when the user fullname lookup fails', function() {
+        beforeEach(async function() {
+          const authentication = proxyquire('./middleware', {
+            '../../common/services/user': userFullNameFailureStub,
           })
 
           await authentication.processAuthResponse()(req, {}, nextSpy)
         })
 
         it('returns next', function() {
```

Comment:
`calls with with error` (although I know it's an existing description but it's doing more than returning next)?

Might also be worth checking that the things that occur after it, don't actually occur as this current test doesn't guarantee that.